src/main/scala/com/atomist/rug/runtime/js/migrations/GraphNodeFunctionsToProperties.scala

```diff
@@ -0,0 +1,5 @@
+package com.atomist.rug.runtime.js.migrations
+
+object NodeNameToProperty extends LiteralJavaScriptReplacementMigration("GraphNode.nodeName is now a property, not a function", "nodeName()", "nodeName")
```

Comment:
I agree! Each time we do a `file/string.contains` - we are adding significant time to each load!